.github/workflows/publish.yml

```diff
@@ -56,3 +56,8 @@ jobs:
         env:
           NODE_ENV: production
           GITHUB_AUTH: ${{ secrets.GITHUB_TOKEN }}
+      - name: Trigger docs release
+        uses: mvasigh/dispatch-action@main
```

Comment:
Is it ok to use 3rd party?

Reply:
yes, anyone creates GitHub Actions these days